MapboxNavigation/RouteMapViewController.swift

```diff
@@ -250,7 +250,7 @@ class RouteMapViewController: UIViewController {
         }
     }
 
-    func notifyAlertLevelDidChange(routeProgress: RouteProgress) {
+    func shouldGiveVoiceInstruction(routeProgress: RouteProgress) {
```

Comment:
RouteMapViewController has no reason to worry about speech. Rename this method to reflect what it does: `updateMapOverlays(for:)`.